runtime/server/queries_timeseries_test.go

```diff
@@ -457,16 +471,19 @@ func TestServer_Timeseries_timezone_dst_forward(t *testing.T) {
 
 	require.NoError(t, err)
 	results := response.GetRollup().Results
-	require.Equal(t, 4, len(results))
+	require.Equal(t, 5, len(results))
 
 	require.Equal(t, "2023-03-26 00:00:00", results[0].Ts.AsTime().Format(time.DateTime))
 	require.Equal(t, 1.0, results[0].Records.Fields["count"].GetNumberValue())
 	require.Equal(t, "2023-03-26 01:00:00", results[1].Ts.AsTime().Format(time.DateTime))
-	require.Equal(t, 1.0, results[1].Records.Fields["count"].GetNumberValue())
-	require.Equal(t, "2023-03-26 02:00:00", results[2].Ts.AsTime().Format(time.DateTime))
+	require.Equal(t, 0.0, results[1].Records.Fields["count"].GetNumberValue())
+	require.Equal(t, "2023-03-26 01:00:00", results[2].Ts.AsTime().Format(time.DateTime))
 	require.Equal(t, 1.0, results[2].Records.Fields["count"].GetNumberValue())
```

Comment:
It looks like the same time series value occurred twice in the output?

Reply:
Yes, during backward shift one hour is double in the local time, should we merge them?

Reply:
Yes, the line chart is expecting one value per time bin